src/utils/backend.js

```diff
@@ -1,22 +1,24 @@
 // eslint-disable-next-line import/no-cycle
-import store from '../store';
+import { EventBus } from './eventBus';
 
 const wrapTry = async (promise) => {
   try {
     return promise.then((res) => {
       if (!res) {
-        store.commit('setBackendStatus', false);
+        EventBus.$emit('maintenance');
         return null;
       }
-      store.commit('setBackendStatus', true);
-      if (!res.ok) throw new Error(`Request failed with ${res.status}`);
+      // store.commit('setBackendStatus', true);
```

Comment:
When does the maintenance page return back to the main feed, if you comment this out?

Reply:
I will remove that comment. It works with eventBus now:
next line is `EventBus.$emit('maintenance');`
and at the App.vue i have a handler:
```js
EventBus.$on('maintenance', () => this.$router.push({ name: 'maintenance' }));
```

Reply:
so basically it's:
https://github.com/aeternity/superhero-ui/blob/f55bc1918c0e51011772c999b653dae9bef68ae7/src/utils/backend.js#L8
and
https://github.com/aeternity/superhero-ui/blob/f55bc1918c0e51011772c999b653dae9bef68ae7/src/App.vue#L56

Reply:
And how the user gets redirected back to the feed when the maintenance is finished?

Reply:
> When does the maintenance page return back to the main feed, if you comment this out?

@mradkov right, need to solve that

Reply:
@mradkov not sure that it's possible now

Reply:
@mradkov I guess I solved that. Need to recheck.

Reply:
@mradkov I don't like all this PR at all but I have no idea how to easily achieve the SSR (remove import store explicitly) 
This should work as a temporary solution. It also redirects to the feed page with the last commit https://github.com/aeternity/superhero-ui/pull/1003/commits/6e1db153b3e122c5234b1af198d47040edfa4687